config/env/default.js

```diff
@@ -28,5 +28,5 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: './modules/core/client/img/brand/favicon.ico'
```

Comment:
@codydaig I should have clarified my response. That's the only place I see that actually leverages that locals variables. I didn't know if you saw it somewhere else.

@rodp82 I think that one change, is all we need. It will be much better to not have that hard-coded.

@codydaig WDYT?